awx/api/serializers.py

```diff
@@ -2515,17 +2554,26 @@ def validate_extra_vars(self, value):
 
     def get_summary_fields(self, obj):
         summary_fields = super(JobTemplateSerializer, self).get_summary_fields(obj)
-        if 'pk' in self.context['view'].kwargs and self.version > 1:  # TODO: remove version check in 3.3
+        if self.is_detail_view and self.version > 1:  # TODO: remove version check in 3.3
```

Comment:
Note to self:  this version check probably needs to move *down* in this function so that we still provide `credential` and `vault_credential` on `/api/v1/`.  The `> 1` check should only be used to determine inclusion of the `extra_credentials` and `credentials` keys (which are not in `/api/v1/`).